packages/client/src/sw.init.ts

```diff
@@ -0,0 +1,18 @@
+export function startServiceWorker() {
+  if ('serviceWorker' in navigator) {
+    window.addEventListener('load', () => {
+      navigator.serviceWorker
+        .register('/sw.js')
+        .then(registration => {
+          console.log(
+            'ServiceWorker registration successful with scope: ',
+            registration.scope
+          )
+        })
+        .catch((error: string) => {
+          console.log('error', error)
+          console.log('ServiceWorker registration failed: ', error)
```

Comment:
_Надо исправить перед влитием_

Оставь тут 1 консоль, если хочешь сообщать пользователям об ошибке, но переделай её в `error`
```suggestion
          console.error('ServiceWorker registration failed: ', error)
```